src/utils/components/UserCredentials/useSSHCommand.ts

```diff
@@ -34,7 +34,7 @@ const useSSHCommand = (
 
   if (user) command += `${user}@`;
 
-  command += `${apiHostname || consoleHostname} -p ${sshServicePort}`;
+  command += `${nodeInternalIPAddress || consoleHostname} -p ${sshServicePort}`;
```

Comment:
```suggestion
  command += `${consoleHostname} -p ${sshServicePort}`;
```

the bug is to use "console host name" instead of "api name"
the internal ip is actually a bug, because a user can't access internal ip addresses

Reply:
changed, all redundant code was removed as well